### PR TITLE
YQL-17371: Fix bytea output in pgrun

### PR DESCRIPTION
**Pull request by marsaly79**

### CODE REVIEW

File: ydb/library/yql/tools/pgrun/pgrun.cpp [hunk 1]
```diff
@@ -804,6 +812,64 @@ std::string FormatCell(const TString& data, const TColumn& column, size_t index,
     return fmt::format("{0}{1:<{2}}", delim, data, column.Width);
 }
 
+// from postgres/src/include/utils/builtins.h
+extern "C" ui64 hex_encode(const char *src, size_t len, char *dst);
```
Comment (vitstn) [inline ydb/library/yql/tools/pgrun/pgrun.cpp:816]: explicit header?

Comment (vitstn) [reply ydb/library/yql/tools/pgrun/pgrun.cpp:816]: or, better, add such function to pg_wrapper/interface/utils

**Review by vitstn — COMMENTED** [review]
please fix comments

File: ydb/library/yql/tools/pgrun/pgrun.cpp (hunk 1 above)
Comment (marsaly79) [reply ydb/library/yql/tools/pgrun/pgrun.cpp:816]: > explicit header?

original PG's header pulls in tons of PG staff we don't need in pgrun

Comment (marsaly79) [reply ydb/library/yql/tools/pgrun/pgrun.cpp:816]: > or, better, add such function to pg_wrapper/interface/utils

You mean moving the declaration of hex_encode() into pg_wrapper/interface/utils.h ?

**Review by vitstn — APPROVED** [review]

**Pull request merged**